### PR TITLE
Create_home_page

### PR DESCRIPTION
**Pull request by s2jack**
Add gitignore
Add linters.yml
Add linters
Test linters
Add ReadMe.md
Add index.html
Add styles.css
Add test.md file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  <head>
+    <link rel="stylesheet" href="styles.css"/>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Hello Microverse</title>
+  </head>
+  <body>
+    <main>
+      <h1>Hello Microverse</h1>
```
Comment (usorfaitheloho) [inline index.html]: According to the project requirement, you are to add a class to your `header` and then style the class in your css file, please add a class in the `h1` tag and then go ahead with the styling in the css file 😄 

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @s2jack  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️

- [ ] Kindly ensure that the Pull Request should have a descriptive title telling you what changes are inside.

> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  <head>
+    <link rel="stylesheet" href="styles.css"/>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Hello Microverse</title>
+  </head>
+  <body>
+    <main>
+      <h1 class="header">Hello Microverse</h1>
```
Comment (wickathou) [inline index.html:12]: **Optional:** consider using class names that are unique. In this case, you created a class `header` which could be confused with `<header>` tags.

You could use instead a class `title` for example.

**Review by wickathou — APPROVED** [review]
Hi @s2jack , 

### It makes me happy to announce

## Your project is complete!

# Congratulations! 🎉 

## Highlights
- [x] Linters working flawlessly :1st_place_medal: 
- [x] Implemented all changes requested :trophy:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**